src/main/java/com/igot/cb/designation/service/impl/DesignationServiceImpl.java

```diff
@@ -249,5 +348,136 @@ private Date parseDate(String value) throws Exception {
     SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
     return dateFormat.parse(value);
   }
+  public ApiResponse readDesignation(String Id) {
+    ApiResponse response = new ApiResponse();
+    try {
+      StringBuilder strUrl = new StringBuilder(cbProperties.getKnowledgeMS());
+      strUrl.append(cbProperties.getOdcsTermCrete()).append("/").append(Id).append("?framework=")
```

Comment:
why are we using cbProperties.getOdcsTermCreate() method here ??
it suppose to read right ?